recipes/assimp/5.0.x/conanfile.py

```diff
@@ -112,14 +111,61 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+    @property
+    def _depends_on_kuba_zip(self):
+        return self.options.with_3mf_exporter
+
+    @property
+    def _depends_on_poly2tri(self):
+        return self.options.with_blend or self.options.with_ifc
+
+    @property
+    def _depends_on_rapidjson(self):
+        return self.options.with_gltf or self.options.with_gltf_exporter
+
+    @property
+    def _depends_on_zlib(self):
+        return self.options.with_assbin or self.options.with_assbin_exporter or \
+               self.options.with_assxml_exporter or self.options.with_blend or self.options.with_fbx or \
+               self.options.with_q3bsp or self.options.with_x or self.options.with_xgl
+
+    def requirements(self):
+        # TODO: unvendor others libs:
+        # - clipper (required by IFC importer): can't be unvendored because CCI
+        #   has 6.4.2, not API compatible with 4.8.8 vendored in assimp
```

Comment:
Should cci start with adding `provides = "clipper"` to a clipper recipe?
Then conan can detect and throw when two recipes provide `clipper`.

Reply:
Eventually yes, but maybe overkill for the moment. If Assimp is shared, clipper is hidden.